### PR TITLE
Edit vignettes

### PR DESCRIPTION
**Pull request by skadauke**
# Description

Edits to the README and vignettes

### CODE REVIEW

**Comment by skadauke** [comment]
Hi, I'm reviewing the vignettes, and I noticed the following:

image

Is this still true? I thought we got rid of this? Are we throwing a warning when we detect this?

Also, I don't think the following is accurate? Rather, REDCap doesn't actually allow you to configure a repeating instrument in a repeating event, correct?

image

**Comment by rsh52** [comment]
> Hi, I'm reviewing the vignettes, and I noticed the following:
> 
> image
> 
> Is this still true? I thought we got rid of this? Are we throwing a warning when we detect this?

This is true, the [`check_repeat_and_nonrepeat`](https://github.com/CHOP-CGTInformatics/REDCapTidieR/blob/a2706d116c0d37c8a3b46bb27cb3e35bc35e0c7d/R/checks.R#L113) function is dedicated to detecting it. We still want this to exist since it should never happen.
 
> Also, I don't think the following is accurate? Rather, REDCap doesn't actually allow you to configure a repeating instrument in a repeating event, correct?
> 
> image

Correct, this isn't possible to do from the REDCap UI. I thought we had a check for this regardless, but I might be mistaken.

File: vignettes/diving_deeper.Rmd [hunk 1]
```diff
@@ -178,13 +182,13 @@ hematology |>
   rmarkdown::paged_table()
 ```
 
-Note that both `physical_exam` and `hematology` are **nonrepeating** instruments. REDCapTidieR **only** allows nonrepeating instruments to occur in repeating events. It will throw an error if it detects a repeating instrument associated with a repeating event.
+Note that REDCapTidieR allows for an instrument to be associated with both repeating and nonrepeating events at the same time. In this case, the the `redcap_event_instance` column will be `NA` in rows that correspond with data captured as part of a nonrepeating event.
```
Comment (rsh52) [inline vignettes/diving_deeper.Rmd:185]: I think we still need to clarify that this is only allowed for nonrepeating instruments, this sounds like it's saying any instrument can be associated with both event structures.

**Review by rsh52 — COMMENTED** [review]
Generally looks good, one comment. 👍

File: vignettes/diving_deeper.Rmd (hunk 1 above)
Comment (skadauke) [reply vignettes/diving_deeper.Rmd:185]: Hm, I think we're making this point at least 3 times in other places. Adding it here I think will make things less clear.

Comment (rsh52) [reply vignettes/diving_deeper.Rmd:185]: I see no harm in a 4th for consistency, up to you though 👍

**Pull request merged**